src/Spd.Manager.Licence/ControllingMemberCrcAppValidation.cs

```diff
@@ -7,74 +7,200 @@
 using System.Threading.Tasks;
 
 namespace Spd.Manager.Licence;
-public class ControllingMemberCrcAppValidator : AbstractValidator<ControllingMemberCrcAppSubmitRequest>
+public class ControllingMemberCrcAppAnonymousSubmitRequestValidator : AbstractValidator<ControllingMemberCrcAppSubmitRequest>
 {
-    public ControllingMemberCrcAppValidator()
+    public ControllingMemberCrcAppAnonymousSubmitRequestValidator()
     {
-        RuleFor(r => r.ParentBizLicApplicationId).NotEmpty();
+        RuleFor(r => r.ParentBizLicApplicationId).NotEqual(Guid.Empty);
+        RuleFor(r => r.WorkerLicenceTypeCode)
+            .Equal(WorkerLicenceTypeCode.SECURITY_BUSINESS_LICENCE_CONTROLLING_MEMBER_CRC);
         RuleFor(r => r.Surname)
             .MaximumLength(40)
+            .NotNull()
             .NotEmpty();
```

Comment:
I've used the combination of `NotNull()` and `NotEmpty()` instead of using is string function

Reply:
that's fine